### PR TITLE
Handle unicode docstrings

### PR DESCRIPTION
**Pull request by analogue**
Fix for unicode docstrings in issue #25.

### CODE REVIEW

**Comment by DanielJomphe** [comment]
Also LGTM after some basic testing. :+1:

File: bravado_core/docstring.py [hunk 1]
```diff
@@ -129,18 +128,18 @@ def create_param_docstring(param_spec):
...
-    s += "\n"
+        s += ' (optional)'
+    s += '\n'
 
     if location == 'body':
         param_type = formatted_type(param_spec.get('schema'))
     else:
         param_type = param_spec.get('type')
-    s += ":type {0}: {1}\n".format(name, param_type)
+    s += ':type {0}: {1}\n'.format(name, param_type)
```
Comment (prat0318) [inline bravado_core/docstring.py]: I guess even this would need a `u` prefix.

**Comment by prat0318** [comment]
lgtm.

**Comment by dnephin** [comment]
lgtm

File: bravado_core/docstring.py (hunk 1 above)
Comment (analogue) [inline bravado_core/docstring.py]: ack

**Pull request merged**